QuizWhois/Common/Models/Base/BaseModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Common.Models.Base
+{
+    public abstract class BaseModel
+    {
+        public Guid Id { get; set; }
```

Comment:
If we want to use guid as a key in db, we should decorate it with [Key] attribute

Reply:
It is not necessary, naming conventions work fine. https://docs.microsoft.com/en-us/ef/core/modeling/keys?tabs=data-annotations